src/sentry/static/sentry/app/views/organizationStats/index.jsx

```diff
@@ -267,7 +261,7 @@ const OrganizationStats = React.createClass({
                 <StackedBarChart
                   points={this.state.orgStats}
                   height={150}
-                  barClasses={['accepted', 'rate-limited', 'black-listed']}
+                  barClasses={['accepted', 'rate-limited']}
```

Comment:
Because the longer goal is show another stat here, and these filtered metrics never really made sense here. I just shoved it here long ago since we had no better place.

Next steps will be surfacing a new metric for "errors" on the org stats page probably under another tab. If we want to show filtered stats here, I'd probably bring it back as under another tab too. My reasoning being, since filtered events don't count against quotas or anything like that, it's not super valuable to show here. I'd rather bring the data local to where you'd care. Or break it out into it's own graph since it's unrelated imo.